### ISSUE TITLE
Create Dockerfile

### ISSUE THREAD

**Issue by cwhits** [MukurtuCMS/mukurtucms#109]
It'd be awesome if there was a dockerfile that would allow Mukurtu to be run in Docker. It'd make it really simple for setting up, upgrades, etc. Since there's already an official Drupal image you could probably base it right off that.

**Comment by steve-taylor-wsu** [comment]
I agree that would be very cool, and we've had other people express similar interest. For the immediate future everybody is tied up working on the upcoming releases and events. I'm hoping we can circle back around and do some work to create/document how to get Mukurtu running on different common platforms. Docker could definitely be part of that. Thanks.
